runtime/ftplugin/sh.vim

```diff
@@ -31,8 +31,21 @@ if has("gui_win32")
 		\	"All Files (*.*)\t*.*\n"
 endif
 
+if executable('bash')
+  " linting
+  setlocal makeprg=bash\ -n\ --\ %:S
+  setlocal errorformat=%f:\ line\ %l:\ %m
+  " keyword lookup
+  if executable('less')
```

Comment:
`executable` tests are expensive and lead to slow experiences. But I don't think `less` is really necessary for `bash -c help` or `man`. The former is short; the latter already uses `$PAGER`.

In general I think it would save some effort to avoid less, or at least to just use the users `$PAGER`.